spec/features/posts_index_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :system do
+  before do
```

Comment:
- [ ] As requested by the previous reviewer, please make sure that you write a Test that asserts you can see a section for pagination if there are more posts than fit on the view. :pray: